library/std/src/sys/sgx/abi/mod.rs

```diff
@@ -45,7 +45,8 @@ unsafe extern "C" fn tcs_init(secondary: bool) {
         // We need to wait until the initialization is done.
         BUSY => {
             while RELOC_STATE.load(Ordering::Acquire) == BUSY {
-                core::arch::x86_64::_mm_pause()
+                // SAFETY: Processor intrinsic
```

Comment:
Is the pause instruction always safe to emit? Maybe this could use core::hint::spin_loop instead?